spec/services/events/list_spec.rb

```diff
@@ -0,0 +1,15 @@
+describe 'Events::List' do
+  describe 'default scope' do
+    it 'orders by latest first by default' do
+      expect(Events::List.new.scope.to_sql).to include(%(ORDER BY "events"."happened_at" DESC))
+    end
+  end
+
+  describe '.for_teacher' do
+    it 'selects only events with a teacher_id matching the provided teacher' do
```

Comment:
My personal preference wording preference would be:

```suggestion
    it 'only selects events with a teacher_id matching the provided teacher' do
```

But not too fussed